pdf2image/pdf2image.py

```diff
@@ -196,6 +197,9 @@ def convert_from_path(
     if auto_temp_dir:
         shutil.rmtree(output_folder)
 
+    if len(images) == 0:
```

Comment:
How about `first_page - last_page + 1` instead of 0? That would catch strange edge cases where some pages would be missing.